lib/Elastica/Transport/Memcache.php

```diff
@@ -79,4 +94,17 @@ public function exec(Request $request, array $params)
 
         return $response;
     }
+
+    /**
+     * Check if key that will be used dont exceed 250 symbols
+     *
+     * @throws Elastica\Exception\Connection\MemcacheException If key is too long
+     * @param  string                                          $key
+     */
+    private function checkKeyLength($key)
```

Comment:
Sure, fixed.
Speaking of that, this library convention is slightly different than that [PSR-2](https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-2-coding-style-guide.md) says:
> Method names SHOULD NOT be prefixed with a single underscore to indicate protected or private visibility.
> Property names SHOULD NOT be prefixed with a single underscore to indicate protected or private visibility.